### PR TITLE
Apply the cleaned up build config to the sdk

### PR DESCRIPTION
**Pull request by nickrum**
@joselcvarela Would be nice if you could check whether this is working for you in Nuxt.

### CODE REVIEW

File: packages/sdk/index.mjs [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './dist/sdk.cjs.js';
```
Comment (rijkvanzanten) [inline packages/sdk/index.mjs:1]: @nickrum Same problem here as with the format-title one?

```suggestion
import DirectusSDK from './dist/sdk.cjs.js';
export default DirectusSDK;
```

Comment (nickrum) [reply packages/sdk/index.mjs:1]: The SDK only has named exports, so this should be fine (I actually copied over this file from the sdk to format-title, that's how the issue with the format-title one snuck in).

**Review by joselcvarela — APPROVED** [review]

**Review by joselcvarela — APPROVED** [review]
Looks good

**Review by joselcvarela — APPROVED** [review]
Haha, I can approve this many consecutive times 😅

**Pull request merged**